src/Resources/skeleton/test/Functional.tpl.php

```diff
@@ -2,9 +2,13 @@
 
 namespace <?= $namespace; ?>;
 
+<?php if($panther_is_available) { ?>
+use Symfony\Component\Panther\PantherTestCase;
+<?php } else { ?>
 use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+<?php } ?>
```

Comment:
Restructure the if statement like this: https://github.com/symfony/maker-bundle/blob/master/src/Resources/skeleton/form/Type.tpl.php#L5-L7

So, `<?php if($panther_is_available): ?>` and similar changes on all the lines

Reply:
done !  
[i also fixed](https://github.com/symfony/maker-bundle/pull/417/commits/367d604345c45b1ae1faee0b83b573c52cd11783) the other `if` statement to follow the same style.